src/PKSim.Core/Snapshots/Mappers/IndividualMapper.cs

```diff
@@ -84,11 +84,15 @@ private async Task convertMoleculesToExpressionProfiles(SnapshotIndividual indiv
             x.Molecule = x.Name;
          });
 
-         var expressionProfiles = await _expressionProfileMapper.MapToModels(individualSnapshot.Molecules);
+         var expressionProfiles = await _expressionProfileMapper.MapToModels(expressionProfilesSnapshot);
          foreach (var expressionProfile in expressionProfiles)
          {
-            var (molecule, individual) = expressionProfile;
-            project.AddBuildingBlock(expressionProfile);
+            var (_, individual) = expressionProfile;
+            //Expression profile may have been added already to the project when converting a project with population 
+            //and individual. So we only add if not available already
+            if (project.BuildingBlockByName<Model.ExpressionProfile>(expressionProfile.Name) == null)
```

Comment:
@Yuri05 This is the fix, I was loading in this case the exp profile twice... => error